### ISSUE TITLE
build failure

### ISSUE THREAD

**Issue by sebastien-bouffier** [VCVRack/Rack#392]
OS (if applicable): W10
Version (or "dev" if compiling from source): dev

Hello Andrew, just pulled the last commit and I have this error 

```
make
g++ -Iinclude -Idep/include -Idep/lib/libzip/include -MMD -g -O3 -march=nocona -                                 ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_WIN                                  -D_USE_MATH_DEFINES -Wsuggest-override -std=c++11 -c -o build/src/main.cpp.o src                                 /main.cpp
g++ -Iinclude -Idep/include -Idep/lib/libzip/include -MMD -g -O3 -march=nocona -                                 ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_WIN                                  -D_USE_MATH_DEFINES -Wsuggest-override -std=c++11 -c -o build/src/plugin.cpp.o s                                 rc/plugin.cpp
In file included from C:/msys64/mingw64/x86_64-w64-mingw32/include/windows.h:71:                                 0,
                 from src/plugin.cpp:15:
include/util.hpp:137:2: error: expected identifier before numeric constant
  ERROR
  ^
include/util.hpp:137:2: error: expected '}' before numeric constant
include/util.hpp:137:2: error: expected unqualified-id before numeric constant
In file included from include/widgets.hpp:10:0,
                 from include/app.hpp:4,
                 from src/plugin.cpp:24:
include/util.hpp:141:10: error: variable or field 'log' declared void
 void log(LogLevel level, const char *format, ...);
          ^~~~~~~~
include/util.hpp:141:10: error: 'LogLevel' was not declared in this scope
include/util.hpp:141:10: note: suggested alternative:
include/util.hpp:134:6: note:   'rack::LogLevel'
 enum LogLevel {
      ^~~~~~~~
include/util.hpp:141:26: error: expected primary-expression before 'const'
 void log(LogLevel level, const char *format, ...);
                          ^~~~~
include/util.hpp:141:46: error: expected primary-expression before '...' token
 void log(LogLevel level, const char *format, ...);
                                              ^~~
include/util.hpp:144:1: error: expected declaration before '}' token
 } // namespace rack
 ^
make: *** [compile.mk:54: build/src/plugin.cpp.o] Error 1
```
best regards

**Comment by phdsg** [comment]
yep, here too. i commented on the commit already.

**Comment by AndrewBelt** [comment]
Using `log()` is also a dumb idea. Fixed with 85c75ee. Even if there are variables and functions called `info`, `warn`, or `fatal`, a name clash in C++ is okay.
